public/apps/configuration/utils/resource-utils.tsx

```diff
@@ -23,3 +26,7 @@ export function generateResourceName(action: string, sourceResourceName: string)
       return '';
   }
 }
+
+export function getResourceUrl(endpoint: string, resourceName: string) {
+  return endpoint + encodeURI(resourceName);
```

Comment:
I suppose the `resourceName` here is the resource id, right?

Shall we use `encodeURIComponent` instead of `encodeURI`?

Reply:
Yep updated.